tests/test_e2e_15_mef_eline.py

```diff
@@ -221,3 +221,158 @@ def test_001_create_update_with_constraints(self):
         failover_path_ids = {link["id"] for link in data["failover_path"]}
         assert current_path_ids == blue_link_ids, current_path_ids
         assert failover_path_ids == red_link_ids, failover_path_ids
+
+    def test_003_evc_vlan_allocation(self):
+        """Test patch an evc with a duplicated tag value"""
+        evc_1 = {
+            "name": "EVC_1",
+            "enabled": True,
+            "uni_a": {
+                "tag": {"tag_type": 1, "value": 100},
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+            },
+            "uni_z": {
+                "tag": {"tag_type": 1, "value": 100},
+                "interface_id": "00:00:00:00:00:00:00:02:2",
+            }
+        }
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, json=evc_1)
+        assert response.status_code == 201, response.text
+        assert 'circuit_id' in response.json()
+
+        # Verify if EVC tag has been allocated
+        topo_url = KYTOS_API + "/topology/v3/interfaces/tag_ranges"
+        response = requests.get(topo_url)
+        data = response.json()
+        actual = data["00:00:00:00:00:00:00:02"]["1"]["available_tags"]["vlan"]
```

Comment:
- If you could also assert `"00:00:00:00:00:00:00:02:2"` `available_tags` that'd be great. 
- Also if you could take the opportunity to also assert the default `tag_ranges` in the response here it'd be great too

btw I've just asked a minor change on `topology`'s PR https://github.com/kytos-ng/topology/pull/166/files#r1344458662

Reply:
Updated this part due to changes in `topology`'s PR.
Added confirmation of changes for `"00:00:00:00:00:00:00:02:2"` in `available_tags` and `tag_ranges` as well as in the other used interfaces.
Added used branches in the top comment.